operator/v1/types_etcd.go

```diff
@@ -28,12 +28,34 @@ type Etcd struct {
 
 type EtcdSpec struct {
 	StaticPodOperatorSpec `json:",inline"`
+	// profile allows user to change the etcd tuning profile which configures
+	// the latency parameters for heartbeat interval and leader election timeouts
+	// allowing the cluster to tolerate longer round-trip-times between etcd members.
+	// Valid values are "Default" and "Slower".
+	// The current default is Default.
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default:=Default
+	// +openshift:enable:FeatureSets=CustomNoUpgrade;TechPreviewNoUpgrade
+	// +optional
+	Profile Profile `json:"profile,omitempty"`
```

Comment:
don't omitempty.  Our configuration APIs show the empty so people know what options are available.